docs/open-telemetry.asciidoc

```diff
@@ -416,4 +416,4 @@ Here is an example of AWS Lambda Node.js function managed with Terraform and the
 [[open-telemetry-otlp-limitations]]
 ===== OpenTelemetry Line Protocol (OTLP)
 
-* The https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp[OTLP/HTTP protocol] is not yet supported by Elastic, use https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc[OTLP/gRPC protocol]. If architecture constraints require to use HTTP rather than gRPC on some segments of the network, please consider deploying an OpenTelemetry Collector to convert OTLP/HTTP to OTLP/gRPC to send those signals to Elastic.
+* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp[OTLP/HTTP protocol] support is currently experimental, use https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlpgrpc[OTLP/gRPC protocol] in production environments. If architecture constraints require to use HTTP rather than gRPC on some critical segments of the network, please consider deploying an OpenTelemetry Collector to convert OTLP/HTTP to OTLP/gRPC to send those signals to Elastic.
```

Comment:
Experimental seems a bit conservative for this feature. @simitt I feel we can jump straight to beta at least. Any concerns with that?

Reply:
No concerns with `beta`; IMO we could even release as `GA` if we do not find anything troublesome while testing and there are no major TODOs left after this PR.

Reply:
Agreed.

Reply:
Modified in 0bffa7217